samples/dnn/dnn_model_runner/dnn_conversion/requirements.txt

```diff
@@ -7,3 +7,7 @@ torchvision>=0.6.1
 
 tensorflow>=2.1.0
 tensorflow-gpu>=2.1.0
+
+paddlepaddle>=2.0.0
+paddlepaddle-gpu>=2.0.0
+paddle2onnx>=0.5.1
```

Comment:
Please extract the dependencies to separate files. Users that follow TF part of tutorial does not need it.